processing/src/main/java/org/apache/druid/query/select/EventHolder.java

```diff
@@ -32,7 +32,7 @@
  */
 public class EventHolder
 {
-  public static final String timestampKey = "timestamp";
+  public static final String timestampKey = "__time";
```

Comment:
well... it was a "documented feature", so documentation for select query needs to be updated as well and please add release notes in the PR description which need to be added to the release notes where this will go as someone might be using the key "timestamp".

Reply:
Thanks for your reply! ok, done!